contents/javascript/Javascript-promises.md

```diff
@@ -0,0 +1,104 @@
+# Javascript: Promises
+
+Author: Daniel Berzin Chua
+
```

Comment:
You can add a Context title to let readers know the context of Promises in Javascript

Reply:
Do you have an example for this? I'm not quite sure what you mean.

Reply:
What I meant was to have a Context title, like your Introduction title or How to use Promises title.

That way people would instantly be mentally prepped of what to anticipate for your first section. This is a small change and completely optional.